Storage/src/Connection/Rest.php

```diff
@@ -628,6 +629,148 @@ private function sendWithRetry($resource, $method, $args)
             $args
         );
 
+        $requestHash = Uuid::uuid4()->toString();
+        $args['onRetryException'] = function (
+            \Exception $e,
+            $currentAttempt,
+            &$arguments
+        ) use ($requestHash) {
+            // Since we the the last attempt number here, so incrementing it
+            // to get the current attempt count
+            ++$currentAttempt;
+            $this->updateRetryInovationHeaders(
+                $arguments,
+                $requestHash,
+                $currentAttempt
```

Comment:
nit: Let's just send `$currentAttempt + 1` here instead of changing the variable value before.

Reply:
Okay

Changed in [33ba71f](https://github.com/googleapis/google-cloud-php/pull/5741/commits/33ba71f14ba1fd93fbfff19cb498b8e566f0d540)